pilot/pkg/model/context.go

```diff
@@ -230,6 +230,11 @@ type Proxy struct {
 		RDS string
 		LDS string
 	}
+
+	// Set to the list of peer authenticated IDs
+	// ( SANs ). The common case is a single spiffee identity,
+	// but for JWT it can be a 'sub' of any type.
+	PeerAuthnIDs []string
```

Comment:
Can we rename to full PeerIdentity here?

Reply:
Sure. The reason I used AuthnID instead of Identity is subtle - in a LOT of Istio code we use un-authenticated identities, like the namespace or name of the pod (from metadata), and we use broad terms ( like ID, identity ) without context. I would really like to have "Authn" or some other naming convention to identify the fields that are trust-worthy. 

But it's just a name, we can also use extensive comments.